vsutil.py

```diff
@@ -157,12 +162,12 @@ def get_w(height: int, aspect_ratio: float = 16 / 9, only_even: bool = True) ->
     """
     width = height * aspect_ratio
     if only_even:
-        width = round(width / 2) * 2
-    return int(round(width))
+        return round(width / 2) * 2
```

Comment:
Same here.

Reply:
Sounds like YAGNI. I can understand the first comment because that is just the last step of a longer process, but here?

Reply:
True.